starter/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/pborman/uuid"
+	"go.temporal.io/sdk/client"
+
+	subs "github.com/afitz0/temporal-subscriptions-go"
+	"github.com/afitz0/temporal-subscriptions-go/zapadapter"
+)
+
+func main() {
+	c, err := client.NewLazyClient(client.Options{
```

Comment:
```suggestion
	c, err := client.Dial(client.Options{
```

We usually encourage users to eagerly connect